app/models/invenio_rdm_record_converter.rb

```diff
@@ -88,6 +88,8 @@ def invenio_metadata(gf)
       "creators": creators(gf.creator),
       "title": gf.title.first,
       "additional_titles": gf.title.last(gf.title.size-1).map{ |title| {"title": title, "type": "alternative_title", "lang": "eng"} },
+      "description": gf.description.first,
+      "additional_descriptions": gf.description.last(gf.description.size-1).map{ |add_desc| {"description": add_desc, "type": "other", "lang": "eng"} },
```

Comment:
I think leaving it as "other" to begin with will be fine. It's seemingly a required field so we need to put something, but really this will need to be looked over by a librarian I think.

Reply:
This is an odd one, the way descriptions are stored. Anyway, what if there is only one description stored? Will this break?

Reply:
I'll check, but I think that `["entry"].last(0)` returns `[]` which is then `[].map{}` which again returns `[]`

Reply:
Sorry, forgot to reply. What I said above is the case.